cloud/pkg/cloudhub/servers/httpserver/nodetask/checknodename.go

```diff
@@ -0,0 +1,43 @@
+package nodetask
```

Comment:
The package name can be changed to `node` or create a new package.

Reply:
Thanks for your suggestion, I will create a new package.